### PR TITLE
add baseguide@1.8.1 w/ git auto-update

### LINKED ISSUE #10015: [Request] Add Baseguide
**Library name:** Baseguide
**Git repository url:** https://github.com/slavanga/baseguide
**npm package url(optional):** https://www.npmjs.com/package/baseguide
**License(s):** MIT
**Official homepage:** http://basegui.de/
**Wanna say something? Leave message here:**
I'm not affiliated with Baseguide, I've just been trying to find a quick & simple alternative to the Bootstrap CSS framework and preferred the look & feel of this one over options like Skeleton, Milligram, and Pure.

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sufuf3**
close #10015, cc @slavanga
@kennynaoh please help me review this PR, thank you.

Pull request for issue: #10015 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/slavanga/baseguide
 * Official website (optional, not the repository):  http://basegui.de
 * NPM package url (optional): https://www.npmjs.com/package/baseguide
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 11
   - Count of stars: 70
   - Count of forks: 8
 * NPM download stats (optional):
   - Downloads in the last day: 16
   - Downloads in the last week: 50
   - Downloads in the last month: 172

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by kennynaoh** [comment]
@sufuf3 Do we need `html5shiv.min.js` and `respond.min.js`?

**Comment by sufuf3** [comment]
@kennynaoh  sorry about that.
And should I add `baseguide-demo.css`?
because it's just a demo.

**Comment by kennynaoh** [comment]
I think we don't need it.

**Comment by slavanga** [comment]
The JS files ```html5shiv.min.js``` and ```respond.min.js``` will be removed in the next major release.
I would not include them.

**Comment by kennynaoh** [comment]
Thanks @slavanga

**Comment by sufuf3** [comment]
@kennynaoh please help me review this PR again, thank you.

File: ajax/libs/baseguide/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "filename": "css/baseguide.min.css",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/slavanga/baseguide.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/baseguide.*"
```
Comment (kennynaoh) [inline ajax/libs/baseguide/package.json]: I think we can use `css/baseguide!(*demo*)`.

**Comment by sufuf3** [comment]
@kennynaoh I modified it.
please help me review this PR again, thank you.

**Comment by kennynaoh** [comment]
I think the reason why you add old versions from git is that some of versions don't exist on npm. You should add the old versions before `v1.3.0`. Thanks!

**Comment by sufuf3** [comment]
@kennynaoh sorry about that I change auto update form git repo.
and I modified the message.

**Comment by sufuf3** [comment]
@kennynaoh I modified it, please help me review this PR again, thank you.

File: ajax/libs/baseguide/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "filename": "css/baseguide.min.css",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/slavanga/baseguide.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "css/baseguide!(*demo*)"
```
Comment (pvnr0082t) [inline ajax/libs/baseguide/package.json]: @sufuf3 We should use `"**/baseguide!(*demo*)"` here.

**Comment by sufuf3** [comment]
@pvnr0082t sorry about that, I modified it, please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I modified it, please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello I modified it, please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@kennynaoh @pvnr0082t  I modified the code of the `baseguide.min.css.map` as following:
 `"file":"main.min.css"` --> `"file":"baseguide.min.css"`

**Comment by PeterDaveHello** [comment]
Since you only add two old versions, please directly mention it in the first line of that commit, and the third line of the second commit message, shorten it or warp it.

Take a look at https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project:
> Wrap it to about 72 characters or so

https://git-scm.com/book/ch5-2.html  
```  
Short (50 chars or less) summary of changes  
  
More detailed explanatory text, if necessary.  Wrap it to  
about 72 characters or so.  In some contexts, the first  
line is treated as the subject of an email and the rest of  
the text as the body.  The blank line separating the  
summary from the body is critical (unless you omit the body  
entirely); tools like rebase can get confused if you run  
the two together.  
  
Further paragraphs come after blank lines.  
  
  - Bullet points are okay, too  
  
  - Typically a hyphen or asterisk is used for the bullet,  
    preceded by a single space, with blank lines in  
    between, but conventions vary here  
```

@cdnjs/intern2 @cdnjs/intern3 please make sure you know that.

**Comment by sufuf3** [comment]
@kennynaoh @pvnr0082t I modified the second commit message.
Please help me review this PR again, thank you.

**Comment by PeterDaveHello** [comment]
Didn't I just give you an example about how to write the second one better? You seem to ignore part of the comments several times.

**Comment by sufuf3** [comment]
@PeterDaveHello sorry about that.
@kennynaoh @pvnr0082t I modified the commit message.
Please help me review this PR again, thank you.

**Pull request merged**